tests/PostmarkClientSuppressionsTest.php

```diff
@@ -0,0 +1,131 @@
+
+        $emailAddress = "test-email@example.com";
+        $suppressionChanges = array(new SuppressionChangeRequest($emailAddress));
+
+        $result = $client->createSuppressions($suppressionChanges);
+
+        $this->assertEquals($emailAddress, $result->Suppressions[0]->EmailAddress);
+        $this->assertEquals("Suppressed", $result->Suppressions[0]->Status);
+    }
+
+    //reactivate suppression
+    public function testClientCanReactivateSuppressions() {
+        $tk = parent::$testKeys;
+        $client = new PostmarkClient($tk->WRITE_TEST_SERVER_TOKEN, $tk->TEST_TIMEOUT);
+
+        $emailAddress = "test-email@example.com";
+        $suppressionChanges = array(new SuppressionChangeRequest($emailAddress));
+
+        $messageStream = "outbound";
+
+        $result = $client->deleteSuppressions($suppressionChanges, $messageStream);
+
+        $this->assertEquals($emailAddress, $result->Suppressions[0]->EmailAddress);
+        $this->assertEquals("Deleted", $result->Suppressions[0]->Status);
+    }
+
+    //invalid request returns failed Status
+    public function testInvalidSuppressionChangeRequestReturnsFailedStatus() {
+        $tk = parent::$testKeys;
+        $client = new PostmarkClient($tk->WRITE_TEST_SERVER_TOKEN, $tk->TEST_TIMEOUT);
+
+        $emailAddress = "invalid-email";
+        $suppressionChanges = array(new SuppressionChangeRequest($emailAddress));
+
+        $messageStream = "outbound";
+
+        $result = $client->createSuppressions($suppressionChanges, $messageStream);
+
+        $this->assertEquals($emailAddress, $result->Suppressions[0]->EmailAddress);
+        $this->assertEquals("Failed", $result->Suppressions[0]->Status);
+    }
+
+    //multiple requests are supported
+    public function testClientCanCreateMultipleSuppressions() {
+        $tk = parent::$testKeys;
+        $client = new PostmarkClient($tk->WRITE_TEST_SERVER_TOKEN, $tk->TEST_TIMEOUT);
+
+        $suppressionChanges = array();
+        for($i = 0; $i < 5; $i++) {
+            $emailAddress = "test-email-$i@example.com";
+            $suppressionChanges[] = new SuppressionChangeRequest($emailAddress);
+        }
+
+        $messageStream = "outbound";
+
+        $result = $client->createSuppressions($suppressionChanges, $messageStream);
+
+        $this->assertNotEmpty($result->Suppressions);
+        foreach($result->Suppressions as $suppressionChangeResult){
+            $this->assertEquals("Suppressed", $suppressionChangeResult->Status);
+        }
+    }
+
+    //invalid message stream throws exception
+    public function testInvalidMessageStreamThrowsException() {
+        $tk = parent::$testKeys;
+        $client = new PostmarkClient($tk->WRITE_TEST_SERVER_TOKEN, $tk->TEST_TIMEOUT);
+
+        $emailAddress = "test-email@email.com";
+        $suppressionChanges = array(new SuppressionChangeRequest($emailAddress));
+
+        $messageStream = "invalid-stream";
```

Comment:
```suggestion
        $messageStream = "?invalid-stream?";
```

It's highly unlikely but 'invalid-stream' is a valid stream id that could mess up this test if for some reason someone used it. We don't allow '?' in a stream id it looks like, so might be a way to make this test more bulletproof.

Reply:
The `MessageStream` value is used when composing the URL and having '?' in it will alter the URL and not use the proper endpoint route. (test will fail)

I updated the `streamID` to be a bit more specific. The configured account should only be used for PHP integration tests so we're probably safe.